index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = Object.assign({},
+    require('./rpc/RPC'),
+    require('./0/Transaction'),
```

Comment:
how are you going to handle the conflict once you add 1/Transaction and 1/Issuance>

Reply:
Sorry, the exports are a work in progress, the base RPC will remain exported though (that's all that's implemented in any dependent applications right now). Will be experimenting with this and get back to you.

Reply:
Decided on an experimental structure for now:
`
module.exports = Object.assign({},
    require('./rpc/RPC'),
    {
        FAT0: {
            Transaction: require('./0/Transaction'),
            Issuance: require('./0/Issuance')
        }
    }
);
`